src/hotspot/share/gc/shenandoah/shenandoahConcurrentMark.cpp

```diff
@@ -262,7 +262,7 @@ void ShenandoahConcurrentMark::finish_mark() {
 
   ShenandoahHeap* const heap = ShenandoahHeap::heap();
   heap->set_concurrent_mark_in_progress(false);
-  heap->mark_complete_marking_context();
+  heap->mark_finished();
```

Comment:
Let's not rename this method. Introduce a new method, `ShenandoahHeap::update_live`, and call it every time after `mark_complete_marking_context()` is called.

Reply:
👍